publishing/ui-metadata/README.md

```diff
@@ -0,0 +1,51 @@
+## UI metadata
+
+By convention, a blendable web app must provide a metadata JSON that responds to the `app-name` blending token.
```

Comment:
It was hard to understand for some time for me. It's confusing to say "app-name", I thought this JSON needs to be provided for URIs like `people`, `images`, `signin` while we said above that starting page should be there... I'm not saying it's bad, just looking for a solution that will be easier to understand. Putting `app-name` like this suggests it's a variable where I should've put my app name, not token itself :)